src/compiler/parser.ts

```diff
@@ -8144,26 +8140,42 @@ namespace ts {
                                 }
                                 break;
                             case SyntaxKind.BacktickToken:
-                                if (state === JSDocState.SavingBackticks) {
-                                    state = JSDocState.SavingComments;
-                                }
-                                else {
-                                    state = JSDocState.SavingBackticks;
-                                }
+                                pushState(state === JSDocState.SavingBackticks ? JSDocState.SavingComments : JSDocState.SavingBackticks);
                                 pushComment(scanner.getTokenText());
                                 break;
                             case SyntaxKind.AsteriskToken:
-                                if (state === JSDocState.BeginningOfLine) {
+                                // if there are two ** after a newline, considering it as a pure comment
+                                if (state === JSDocState.BeginningOfLine && lookAhead(() => nextTokenJSDoc() !== SyntaxKind.AsteriskToken)) {
+                                    /*
+                                    * @param x
+                                    * in the first line
+                                    */
+                                    // to check tag starting newline without comments like above
+                                    // it needs to actually encounter a newline
+                                    if (states.length === 1 && lookAhead(() => nextTokenJSDoc() !== SyntaxKind.NewLineTrivia)) {

```

Comment:
nothing ever uses or pops from `states`; the only read is to check that `states.length === 1`. It should be a boolean instead, maybe called `isFirstLine`? That's a guess from reading the code.